src/graphql/consts/pullRequest.ts

```diff
@@ -0,0 +1,29 @@
+import { persons } from "./person";
+import { PullRequest, PullRequestStatus } from "../../generated/types";
```

Comment:
where is this file i dont see it

Reply:
Because they in .gitignore, run `npm run generate`